packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -32,12 +33,12 @@ export const getInputPrice = (
   outputReserve,
   feeBasisPoints = 30,
 ) => {
-  const oneMinusFeeInTenThousandths = subtract(10000, feeBasisPoints);
-  const inputWithFee = multiply(inputValue, oneMinusFeeInTenThousandths);
-  const numerator = multiply(inputWithFee, outputReserve);
-  const denominator = add(multiply(inputReserve, 10000), inputWithFee);
+  const oneMinusFeeInTenThousandths = BigInt(10000) - BigInt(feeBasisPoints);
```

Comment:
'inTenThousandths' sounds wrong to me.  Shouldn't it be `oneMinusFeeTimesTenThousand`?